### ISSUE TITLE
Withdrawal operation

### ISSUE THREAD

**Issue by MathieuDutSik** [AlacrisIO/meta#88]
The withdrawal operation is the only part that is difficult to implement. Depositing money to the bank is not an issue, withdrawing from it is an issue.
The operation goes in two steps:
 - First set a claim withdrawal of some money in the ethereum blockchain. The claim withdrawal must have a confirmed state that it puts as reference.
 - After the end of a fixed period P which is measured in blocks (not time), the user can do the withdrawal.

In other words, the operator can prevent the withdrawal from happening during the period P. During this period the operator can block the withdrawal. This blocking is done by the following way: He has to prove that the states that has been put does not come from the transfer that he is claiming about.
This requires covering a number of cases.
